bin/worker.ts

```diff
@@ -11,6 +14,8 @@ import { DomainEventSubscriberFactoryInterface } from '@standardnotes/domain-eve
 
 const container = new ContainerConfigLoader
 void container.load().then(container => {
+  dayjs.extend(utc)
```

Comment:
this won't be needed if you use the `TimerInterface` implementation - see my other comment.